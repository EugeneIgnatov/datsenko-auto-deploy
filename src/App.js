import React from 'react';
// import logo from './logo.svg';
import TellMeWhy from './components/TellMeWhy';
import './App.css';

function App() {
  // const [care, setCare] = useState(false);
  // const [suffer, setSuffer] = useState(false);

  return (
    <div className='App'>
      <TellMeWhy
      // care={care}
      // setCare={setCare}
      // suffer={suffer}
      // setSuffer={setSuffer}
      />
    </div>
  );
}

export default App;
