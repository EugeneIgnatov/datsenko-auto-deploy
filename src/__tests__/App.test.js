import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

test('renders the corect content ', () => {
  const { container } = render(<App />);

  expect(container.firstChild).toMatchSnapshot();
});
