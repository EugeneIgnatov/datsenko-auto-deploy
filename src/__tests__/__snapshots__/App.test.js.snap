// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the corect content  1`] = `
<div
  class="App"
>
  <div>
    <h1>
      Tell me why you gave me such a hard topic to learn?
    </h1>
    <h3>
      Because I care
    </h3>
    <button
      data-testid="careBtn"
    >
      Care Vote
    </button>
    <h3>
      Because I want you to suffer
    </h3>
    <button
      data-testid="sufferBtn"
    >
      Suffer Vote
    </button>
  </div>
</div>
`;
